build/common.props

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<!--Semantic Version-->
+	<PropertyGroup>
+		<!-- 
+		  Semantic Version. See http://semver.org for full details. 
+		  Update for every public release. 
+		-->
+		<SemanticVersionMajor>3</SemanticVersionMajor>
+		<SemanticVersionMinor>2</SemanticVersionMinor>
+		<SemanticVersionPatch>0</SemanticVersionPatch>
+		
+		<!-- 
+		Date when Semantic Version was changed. 
+		Update for every public release.
+		-->
+		<SemanticVersionDate>2015-8-21</SemanticVersionDate>
+
+		<!-- 
+		Pre-release version is used to distinguish internally built NuGet packages.
+		Pre-release version = Minutes since semantic version was set, divided by 5 (to make it fit in a UInt16).
+		-->
+		<PreReleaseVersion>$([MSBuild]::Divide($([System.DateTime]::Now.Subtract($([System.DateTime]::Parse($(SemanticVersionDate)))).TotalMinutes), 5).ToString('F0'))</PreReleaseVersion>
+	</PropertyGroup>
+	
+	<!--Setting the Pre-release/Build meta-data from CI if Version is set-->
+	<PropertyGroup Condition="'$(Version)' != ''">
+		<PreReleaseVersion>$(Version)</PreReleaseVersion>
+	</PropertyGroup>
+	
+	<!--Not using the build number for FileVersion on release build-->
+	<PropertyGroup Condition="'$(PublicRelease)' == 'true'">
+		<PreReleaseVersion>0</PreReleaseVersion>
+	</PropertyGroup>
+
+  <!-- Generate AssemblyFileVersion and AssemblyVersion attributes. -->
+  <PropertyGroup>
+    <!-- Turn on dynamic assembly attribute generation -->
+    <AssemblyAttributesPath>$(MSBuildProjectDirectory)\obj\$(Configuration)\AssemblyInfo.g.cs</AssemblyAttributesPath>
+    <GenerateAdditionalSources>true</GenerateAdditionalSources>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <!-- 
+      AssemblyVersion and AssemblyFileVersion attributes are generated automatically for every build.
+      NuGet package version is derived from AssemblyFileVersion.
+    -->
+    <AssemblyAttributes Include="AssemblyVersion">
+      <_Parameter1>$(SemanticVersionMajor).$(SemanticVersionMinor).$(SemanticVersionPatch).$(PreReleaseVersion)</_Parameter1>
+    </AssemblyAttributes>
+    <AssemblyAttributes Include="AssemblyFileVersion">
+      <_Parameter1>$(SemanticVersionMajor).$(SemanticVersionMinor).$(SemanticVersionPatch)$(PreReleaseVersion)</_Parameter1>
```

Comment:
When i tried adding -beta to the Assembly version it error out, are we sure it's the assembly version of the assembly on the dnx or the NuGet version?